flaml/searcher/flow2.py

```diff
@@ -237,7 +239,8 @@ def complete_config(self, partial_config: Dict,
         ''' generate a complete config from the partial config input
         add minimal resource to config if available
         '''
-        if self._reset_times: # not the first time, use random gaussian
+        if self._reset_times and partial_config==self.init_config:
```

Comment:
why the second condition is needed? can you add comments?

Reply:
added